index.html

```diff
@@ -294,6 +294,17 @@ <h3 class="grouping-title">Methods</h3>
               from the value configured with the camera-orbit attribute on account
               of user interaction or camera interpolation.</p>
             </li>
+            <li>
+              <div>getFieldOfView()</div>
+              <p>Returns the realtime camera vertical field of view in degrees. 
+              This value may differ from the value configured with the camera-orbit 
+              attribute on account of user interaction or camera interpolation.</p>
+            </li>
+            <li>
+              <div>jumpToGoal()</div>
```

Comment:
I can make the referential leap with `getFieldOfView` to understand that it is referring to the camera (what else could FOV be referring to?). `jumpToGoal` is relatively less clear.

WDYT about `jumpCameraToGoal()`?